packages/flutter/lib/src/widgets/table.dart

```diff
@@ -90,6 +90,10 @@ class _TableElementRow {
 ///
 /// For more details about the table layout algorithm, see [RenderTable].
 /// To control the alignment of children, see [TableCell].
+/// 
```

Comment:
It looks like there may be a trailing white space here that is causing the analyzer to fail.

Reply:
Thank you & that should be resolved.